commons/all/src/test/java/org/infinispan/commons/util/HashMapTest.java

```diff
@@ -21,6 +21,21 @@ public void testCreateHopscotchWithRemovals() {
       testCreateWithRemovals(new HopscotchHashMap<>(32));
    }
 
+   @Test
+   public void testFail() {
+      assertEquals(1,2);
+   }
+
+   public static int i=0;
+   @Test
+   public void testFlaky() {
+      if (i==0) {
+         i=1;
+         assertEquals(1,2);
+      }
+      assertEquals(1,1);
+   }
```

Comment:
Maybe we should move those tests in a different test class. If I understand correctly, the idea here is to simulate failing an flaky tests.

Reply:
yeah I need them just for dev, I'll cleanup in the final version